redwood-lazylayout-schema/src/main/kotlin/app/cash/redwood/lazylayout/widgets.kt

```diff
@@ -25,33 +25,175 @@ import app.cash.redwood.ui.Margin
 
 @Widget(1)
 public data class LazyList(
+  /**
+   * The layout orientation of the list.
+   */
   @Property(1) val isVertical: Boolean,
+
+  /**
+   * Invoked when the user has scrolled the list, such that the `firstVisibleItemIndex` or the
+   * `lastVisibleItemIndex` has changed. When the user performs a fling, [onViewportChanged] will be
+   * invoked multiple times.
+   *
+   * The `firstVisibleItemIndex` is the index of the first partially visible item within the
+   * viewport. The `lastVisibleItemIndex` is the index of the last partially visible item within the
+   * viewport.
+   */
   @Property(2) val onViewportChanged: (firstVisibleItemIndex: Int, lastVisibleItemIndex: Int) -> Unit,
+
+  /**
+   * The number of un-emitted items before the [items] window.
```

Comment:
I don't think it's clear enough what it means to be "emitted" in this context, especially since the term isn't used anywhere else in these doc comments.

Reply:
Good shout! The relationship between the `items` window and emitted items is completely absent. Will add that in.